atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/knowledge/DefaultKnownAbortedTransactions.java

```diff
@@ -62,10 +66,16 @@ public class DefaultKnownAbortedTransactions implements KnownAbortedTransactions
     public static DefaultKnownAbortedTransactions create(
             KnownConcludedTransactions knownConcludedTransactions,
             FutileTimestampStore futileTimestampStore,
-            TaggedMetricRegistry registry) {
+            TaggedMetricRegistry registry,
+            Optional<InternalSchemaInstallConfig> config) {
```

Comment:
seems like this will always be set unless I'm missing something?

Reply:
this was done for testing